lib/served/resource/requestable.rb

```diff
@@ -39,6 +39,10 @@ def initialize
           '{/resource*}{/id}.json{?query*}'
         end
 
+        class_configurable :raise_on_exceptions do
+          true
+        end
+
```

Comment:
this is great for backward compatibility, eventually we probably should make this the default behavior, and switch how `save!` and `save`